etl/steps/export/explorers/agriculture/latest/crop_yields.config.yml

```diff
@@ -33,60 +33,220 @@ dimensions:
+      - slug: avocados
+        name: Avocados
       - slug: bananas
         name: Bananas
       - slug: barley
         name: Barley
       - slug: beans
         name: Beans
+      - slug: blueberries
+        name: Blueberries
+      - slug: broad_beans
+        name: Broad beans
+      - slug: buckwheat
+        name: Buckwheat
+      - slug: cabbages
+        name: Cabbages
+      - slug: carrots_and_turnips
+        name: Carrots and turnips
+      - slug: cashew_nuts
+        name: Cashews
       - slug: cassava
         name: Cassava
+      - slug: cauliflowers_and_broccoli
+        name: Cauliflowers and broccoli
       - slug: cereals
         name: Cereals
+      - slug: cherries
+        name: Cherries
+      - slug: chestnut
+        name: Chestnut
+      - slug: chickpeas
+        name: Chickpeas
+      - slug: chillies_and_peppers
+        name: Chillies and peppers
+      - slug: cinnamon_and_cinnamon_tree_flowers__raw
+        name: Cinnamon
+      - slug: citrus_fruit
+        name: Citrus fruit
+      - slug: cloves__whole_stems__raw
+        name: Cloves
       - slug: cocoa_beans
         name: Cocoa beans
+      - slug: coconuts
+        name: Coconuts
       - slug: coffee_beans
         name: Coffee beans
       - slug: corn__maize
         name: Corn (maize)
       - slug: cotton
         name: Cotton
+      - slug: cow_peas
+        name: Cow peas
+      - slug: cranberries
+        name: Cranberries
+      - slug: cucumbers_and_gherkins
+        name: Cucumbers and gherkins
+      - slug: currants
+        name: Currants
+      - slug: dates
+        name: Dates
+      - slug: eggplants
+        name: Eggplants
+      - slug: figs
+        name: Figs
+      - slug: fruit
+        name: Fruit
+      - slug: garlic
+        name: Garlic
+      - slug: ginger__raw
+        name: Ginger
+      - slug: grapefruit
+        name: Grapefruit
+      - slug: grapes
+        name: Grapes
+      - slug: beans__green
+        name: Green beans
+      - slug: chillies_and_peppers__green__capsicum_spp__and_pimenta_spp_
+        name: Green chillies and peppers
+      - slug: onions_and_shallots__green
+        name: Green onions and shallots
+      - slug: peas__green
+        name: Green peas
       - slug: groundnut
         name: Groundnut
+      - slug: hazelnuts
+        name: Hazelnuts
+      - slug: hempseed
+        name: Hempseed
+      - slug: herbs
+        name: Herbs
+      - slug: kiwi
+        name: Kiwi
+      - slug: leeks
+        name: Leeks
+      - slug: lemons_and_limes
+        name: Lemons and limes
+      - slug: lentils
+        name: Lentils
       - slug: lettuce
         name: Lettuce
+      - slug: linseed
+        name: Linseed
+      - slug: mangoes
+        name: Mangoes
+      - slug: melon
+        name: Melon
       - slug: millet
         name: Millet
+      - slug: mustard_seed
+        name: Mustard seed
       - slug: oats
         name: Oats
       - slug: oil_palm
         name: Oil palm
```

Comment:
Palm oil is perhaps more widely known?

Reply:
I think in this case the crop is "Oil palm".